proto/lambda_protobuf.ml

```diff
@@ -262,6 +264,8 @@ let rec value_from : Types.value -> value = function
      | None -> Fmt.invalid_arg "Cannot unify %a and %a"
                  Lambda.Type.pp t
                  Lambda.Type.pp terror)
+  | Types.Return { value; _ } -> value_from value
+  | Types.Abstract -> invalid_arg "Impossible to un-serialize an abstract value"
```

Comment:
If this is an error, wouldn't it be better to omit values of abstract type altogether?

Reply:
I think it's better to delete it too. However, the server will respond an error if it wants to serialize an abstract type. In this case, we can serialize an abstract type and the client can deal with it differently.